quickstart/201-iot-hub-with-device-provisioning-service/providers.tf

```diff
@@ -0,0 +1,18 @@
+terraform {
+  required_version = ">=0.12"
```

Comment:
The mainstream version of Terraform is 1.3 now, could we adjust this `required_version` to `>=1.0`?